xcvm/lib/core/src/program.rs

```diff
@@ -4,7 +4,9 @@ use scale_info::TypeInfo;
 use serde::{Deserialize, Serialize};
 
 #[cfg_attr(feature = "std", derive(schemars::JsonSchema))]
-#[derive(Clone, PartialEq, Eq, Debug, Encode, Decode, TypeInfo, Serialize, Deserialize)]
+#[derive(
+	Clone, PartialEq, Eq, Debug, Encode, Decode, TypeInfo, Serialize, Deserialize
+)]
 #[serde(rename_all = "camelCase")]
 pub struct Program<Instructions> {
 	pub tag: Option<Vec<u8>>,
```

Comment:
Can we avoid the Option here? Isn't an empty tag semantically the same?

Reply:
We can I guess. Empty vec will be serialized into `[]` and `None` to `null`. But let me check if we do any kind of stuff based on the None check on tag.

Reply:
Has been updated on latest core